src/components/LightSwitch.js

```diff
@@ -1,9 +1,24 @@
-import React from "react"
+import React, { useState } from 'react'
 
-const LightSwitch = () => {
+const LightSwitch = () =>{
+  const [toggleSwitch, setToggleSwitch] = useState("off")
+
+  const [color, setColor] = useState("white")
+
+  const onAndOff = () =>{
+    if (toggleSwitch === "off") {
+      setToggleSwitch("on")
+      setColor("yellow")  
+    } else {
+      setToggleSwitch("off")
+      setColor("white")
+    } 
+  }
```

Comment:
Makes total sense. The conditional (ternary) operator is another option to make build condensed if...else statements. You can read about them [here](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Operators/Conditional_Operator)